pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -86,6 +86,11 @@ func (w writer) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
+func onPodDeletedOrEvicted(pod *corev1.Pod, usingEviction bool) {
+	msg := fmt.Sprintf("pod: %s:%s evicted from node: %s method: %t", pod.ObjectMeta.Namespace, pod.ObjectMeta.Name, pod.Spec.NodeName, usingEviction)
+	klog.Info(msg)
```

Comment:
```suggestion
        verbStr := "Deleted"
        if usingEviction {
            verbStr = "Evicted"
        }
        klog.Info(fmt.Sprintf("%s pod from Node", verbStr),
                  "pod", fmt.Sprintf("%s/%s", pod.Name, pod.Namespace))
```

Reply:
I think I like the original better.
True and false don't clearly relate to Deleted and Evicted

Reply:
changed the log message and force pushed the change into the last commit - so as not to introduce another commit just for the purpose of changing a printf.